lib/ohloh_scm/adapters/hg_adapter.rb

```diff
@@ -1,9 +1,14 @@
 module OhlohScm::Adapters
-	class HgAdapter < AbstractAdapter
-		def english_name
-			"Mercurial"
-		end
-	end
+  class HgAdapter < AbstractAdapter
+    def english_name
+      "Mercurial"
+    end
+
+    def branch_name=(branch_name)
+      branch_name = nil if branch_name.to_s.empty?
```

Comment:
No, blank? will throw `NoMethodError: undefined method 'blank?' for nil:NilClass`